pyproject.toml

```diff
@@ -26,9 +26,14 @@ classifiers = [
 ]
 requires-python = ">= 3.11, < 4"
 dependencies = [
-  "typing-extensions >= 4.5.0, < 5",
-  "frequenz-api-dispatch >= 0.12.0, < 0.13",
-  "frequenz-client-base >= 0.1.0, < 0.2",
+  "typing-extensions >= 4.6.1, < 5",
+# Use marenz fork @ 0d04bc3 until the next release
+  "frequenz-api-dispatch @ git+https://github.com/marenz/frequenz-api-dispatch.git@0d04bc3",
+#  "frequenz-api-dispatch >= 0.12.0, < 0.13",
+# Directly use unreleased/unmerged commit fe4e238 until the next release
+  "frequenz-client-base @ git+https://github.com/marenz/frequenz-client-base-python.git@fe4e238",
+#  "frequenz-client-base >= 0.1.0, < 0.2",
+  "frequenz-sdk == v1.0.0-rc4",
```

Comment:
We shouldn't import the SDK in API clients, the idea is to do it the other way around (the SDK will import the client).